pages/common/asdf.md

```diff
@@ -0,0 +1,28 @@
+# asdf
+
+> Command line interface for managing versions of different packages.
+> Homepage: <https://asdf-vm.com>.
+
+- List all available plugins:
+
+`asdf plugin-list-all`
+
+- Install a plugin:
+
+`asdf plugin-add <name>`
```

Comment:
```suggestion
`asdf plugin-add {{name}}`
```

Please use the appropriate `{{token}}` syntax for user tokens. Same for every other token in the page, not only this one.